coalib/bears/requirements/PackageRequirement.py

```diff
@@ -1,4 +1,5 @@
 from coala_decorators.decorators import generate_eq, generate_repr
+import os
```

Comment:
@Adrianzatreanu coala module imports should come after inbuilt modules. :stuck_out_tongue_winking_eye: